pkg/frontend/clustermanager_type.go

```diff
@@ -0,0 +1,35 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"fmt"
+)
+
+func (f *frontend) readOcmResourceType(vars map[string]string) error {
```

Comment:
This doesn't really "read" anything does it? How about `validateOcmResourceType`?